app/models/job.rb

```diff
@@ -1,5 +1,26 @@
 # frozen_string_literal: true
 
 class Job < ApplicationRecord
-  validates :company, :title, :description, :link, :location, :image_url, presence: true
+  enum sponsorship_level: %i[opal sapphire emerald ruby], _suffix: true
```

Comment:
@stephanieminn This is so cool! Here's what this allows us to do:

```ruby
job = Job.first

job.sponsorship_level # "opal"
job.emerald_sponsorship_level? # false
job.opal_sponsorship_level? # true
job.random_sponsorship_level? # undefined method
job.sponsorship_level = 'emerald' # works
job.sponsorship_level = 'random' # 'random' is not a valid sponsorship level
job.sponsorship_level = 6 # 6 is not a valid sponsorship level
```

I added `_suffix` true so that all of these methods would include the words "sponsorship_level," making it clearer what they're referring to.